### PR TITLE
Signup: Update colors and background

### PR DESCRIPTION
**Pull request by sixhours**
#### Changes proposed in this Pull Request

* Adds a dark blue background color, with white or light gray text
* Adds two fixed shapes to the background
* Both these changes bring us closer to parity with the new WP.com home page and marketing materials

Before:

screen shot 2018-10-12 at 1 41 29 pm

After:

screen shot 2018-10-12 at 1 41 21 pm

#### Testing instructions

* Switch to this PR and navigate to `/start/`
* Check to see if the colors have changed throughout signup
* Check any special flows (ie. `/start/rebrand-cities/`, or select "sell products" at the start)
* Check elsewhere on WP.com to make sure the colors are not affected (things like `/add/domains/` or `/plans/`, for example)

### CODE REVIEW

**Comment by sixhours** [comment]
[Add new colours + background to signup](https://trello.com/c/hVoiZNRh/128-add-new-colours-background-to-signup)

File: assets/stylesheets/shared/_colors.scss [hunk 1]
```diff
@@ -76,3 +76,12 @@ $color-skype: #00AFF0;
 $color-telegram: #0088cc;
 $color-whatsapp: #43d854;
 $color-wordpressOrg: #585c60;
+
+// New Signup Colors
```
Comment (scruffian) [inline assets/stylesheets/shared/_colors.scss]: If we might use these colors elsewhere in the future then "signup" might not be the best prefix. Is there a name for this rebranding we could use here?

**Comment by scruffian** [comment]
Found a few other contrast issues:

screen shot 2018-10-15 at 14 36 41
screen shot 2018-10-15 at 14 36 18

Also I think the borders of these boxes look a little off with this background, but that might just be me.
screen shot 2018-10-15 at 14 35 15
screen shot 2018-10-15 at 14 35 01

**Comment by fditrapani** [comment]
Thanks for putting this together @sixhours. I apologize for this but I'd like for us to hold on this colour update for now. We still have to figure out the transition between signup and Calypso which is still being figured out. The layout and animations would be better sections to prioritize before we get to the colour.

**Comment by sixhours** [comment]
Closing; this is out of date now that we have a new color palette.

**Pull request closed without merging**